src/NzbDrone.Core/Statistics/HistoryStatistics.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Statistics
+{
+    public class HistoryStatistics
```

Comment:
In that case IndexerStatistics would be more applicable. Since we could also have a EpisodeStatistics, which would get data from your series and episodes and say what kind of qualities you have and the amount of data and episodes.